addons/mail/models/res_users.py

```diff
@@ -132,6 +141,30 @@ def write(self, vals):
                         subtype_xmlid='mail.mt_note'
                     )
 
+        if 'login' in vals:
+            self._notify_security_setting_update(
+                _("Security Update: Login Changed"),
+                _("Your account login has been updated"),
+            )
+        if 'password' in vals:
+            self._notify_security_setting_update(
+                _("Security Update: Password Changed"),
+                _("Your account password has been updated"),
+            )
+        if 'email' in vals:
+            # when the email is modified, we want notify the previous address (and not the new one)
+            for user, previous_email in previous_email_by_user.items():
+                self._notify_security_setting_update(
+                    _("Security Update: Email Changed"),
+                    _(
+                        "Your account email has been changed from %s to %s.",
```

Comment:
I would use named parameters :) easier for translations (and Martin will stop asking stuff for digest)